Examples/WildeGuess/WildeGuess/Components/QuoteComponent.h

```diff
@@ -9,7 +9,7 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-#import "CKCompositeComponent.h"
+#import <ComponentKit/CKCompositeComponent.h>
```

Comment:
but if we do convert the example project to using the umbrella header, let's do it 100% (all headers too)